plugins/jquery-filters.php

```diff
@@ -14,16 +14,20 @@
 $sites = jquery_sites();
 $options = array_merge( $options, $sites[ JQUERY_LIVE_SITE ]['options'] );
 foreach ( $options as $option => $value ) {
-	if ( $option === 'stylesheet' || $option === 'template' ) {
-		// Don't mess with themes for now.
-		continue;
-	}
-	if ( $option === 'active_plugins' ) {
-		// Deprecated
-		// In production, Puppet manages activation of per-site plugins.
-		// Locally, global plugins are enabled via mu-plugins/_loader.php,
-		// and per-site plugins are activated by the imported database.
-		continue;
+	// Skip these in production, where they are managed by puppet.
+	// Staging should be allowed to set them for testing.
```

Comment:
I'd add that it is for testing "with an empty database" specifically, since it works fine without it in the way we've been testing so far.

Reply:
I figure it should be allowed either way in case we ever wanted to test theme changes.

Reply:
Right, I didn't think about the "allowed" phrasing. My point is that I foresee future "improvements" removing this because it seems to work. I want to leave a trace that says this enables primarily testing with a clean database, as it literally doens't work without it in that scenario. It can then also be used for other use cases indeed.